aries-site/src/examples/foundation/icons/IconComponentExample.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+import { Menu } from 'grommet';
+import { Add } from 'grommet-icons';
+
+export const IconComponentExample = () => {
+  return <Menu icon={<Add />} label="Add User" />;
```

Comment:
I'm wondering if a normal Button would be a good addition here since it has a visible border. I think having them both would be nice